### PR TITLE
Send user identifiers to server when identify is called

### PR DESCRIPTION
**Pull request by wyattjoe40**
Sends user identifiers to the events endpoint when `identify` is called.

### CODE REVIEW

File: Example/Example/ViewController.m [hunk 1]
```diff
@@ -21,11 +24,12 @@ - (void)viewDidLoad {
...
     sdk = [[ATTNSDK alloc] initWithDomain:@"YOUR_ATTENTIVE_DOMAIN" mode:@"production"];
     
+    
     // Register the current user with the Attentive SDK. Replace "APP_USER_ID"
-    // with the current user's ID. You must register a user ID before calling
-    // `trigger` on a Creative.
+    // with the current user's ID.
     // TODO - more info here
-    [sdk identify:@{ IDENTIFIER_TYPE_CLIENT_USER_ID: @"APP_USER_ID", IDENTIFIER_TYPE_PHONE: @"+14156667777"}];
+    _userIdentifiers = @{ IDENTIFIER_TYPE_CLIENT_USER_ID: @"APP_USER_ID", IDENTIFIER_TYPE_PHONE: @"+14156667777"};
```
Comment (oliviakim321) [inline Example/Example/ViewController.m:31]: not a blocker for this PR at all, but should we show an example here with all the identifiers & add a comment that says specify as many of these identifiers as you can, but they are all optional?

or should we just show the most important identifiers like phone & email? not sure client user ID is the most important one anymore

Comment (wyattjoe40) [reply Example/Example/ViewController.m]: hmmm. I like showing all of the identifiers. I added to this PR. What do you think?

Comment (oliviakim321) [reply Example/Example/ViewController.m]: LGTM!

File: Tests/ATTNAPITest.m [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+}
+
+- (void)testConstructUserIdentityUrl {
+    ATTNAPI* api = [[ATTNAPI alloc] init];
+    ATTNUserIdentity* userIdentity = [[ATTNUserIdentity alloc] initWithIdentifiers:@{IDENTIFIER_TYPE_CLIENT_USER_ID: @"some-client-id", IDENTIFIER_TYPE_EMAIL: @"some-email@email.com"}];
+    
+    NSURL* url = [api constructUserIdentityUrl:userIdentity domain:@"some-domain"];
+    XCTAssertNotNil(url);
```
Comment (oliviakim321) [inline Tests/ATTNAPITest.m:38]: should we compare the full url here? or are we going to write the tests in another PR

Comment (wyattjoe40) [reply Tests/ATTNAPITest.m:38]: I'm thinking tests in another PR 😬

File: Sources/ATTNAPI.h [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+//  attentive-ios-sdk
+//
+//  Created by Wyatt Davis on 11/28/22.
+//
+
+#ifndef ATTNAPI_h
+#define ATTNAPI_h
+
+#import <Foundation/Foundation.h>
+#import "ATTNUserIdentity.h"
```
[the remark this answers is not in the document]
Comment (wyattjoe40) [reply Sources/ATTNAPI.h:12]: interesting, IDK. Reading...

File: Tests/ATTNAPITest.m (hunk 2 above)
Comment (oliviakim321) [reply Tests/ATTNAPITest.m:38]: sgtm!

File: Sources/ATTNAPI.h (hunk 3 above)
[the remark this answers is not in the document]
Comment (wyattjoe40) [reply Sources/ATTNAPI.h]: Nice callout! I didn't know this. Updated, and confirmed it works with the Example app.

**Review by oliviakim321 — APPROVED** [review]

**Pull request merged**